test/functional/tests/fault_injection/test_fault_injection_many_to_one.py

```diff
@@ -226,6 +226,96 @@ def test_one_core_fail(cache_mode):
         core_dev1.plug()
 
 
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core1", DiskTypeLowerThan("cache"))
+@pytest.mark.require_disk("core2", DiskTypeLowerThan("cache"))
+def test_one_core_fail_dirty():
+    """
+        title: Test if OpenCAS correctly handles failure of one of multiple core devices.
+        description: |
+          When one core device fails in a single cache instance all cache lines are dirty and
+          mapped to the core device, other cores due to that are unable to insert andy data and
+          are serviced in pass-through.
+        pass_criteria:
+          - No system crash.
+          - Second core is able to use OpenCAS.
+    """
+    with TestRun.step("Prepare one cache and two core devices."):
+        cache_dev = TestRun.disks["cache"]
+        cache_dev.create_partitions([Size(1, Unit.GibiByte)] * 2)
+        cache_part = cache_dev.partitions[0]
+        core_dev1 = TestRun.disks["core1"]  # This device would be unplugged.
+        core_dev1.create_partitions([Size(2, Unit.GibiByte)])
+        core_part1 = core_dev1.partitions[0]
+        core_dev2 = TestRun.disks["core2"]
+        core_dev2.create_partitions([Size(2, Unit.GibiByte)])
+        core_part2 = core_dev2.partitions[0]
+        Udev.disable()
+
+    with TestRun.step("Start cache"):
+        cache_mode = CacheMode.WB
+        cache = casadm.start_cache(cache_part, cache_mode=cache_mode, force=True)
+        caches_count = len(casadm_parser.get_caches())
+        if caches_count != 1:
+            TestRun.fail(f"Expected caches count: 1; Actual caches count: {caches_count}.")
```

Comment:
I think we prefer to use "exception" instead of "fail" for this kind of errors (error in product not directly related to the tested functionality)

Reply:
.. alternatively you can remove this check entirely, as if there was something wrong with starting the cache an exception would have been risen.